tasks/src/app/simple-button/simple-button.component.html

```diff
@@ -1 +1,3 @@
-<button (click)="increseNumberOfCounts()">Number of clicks {{numberOfClicks}}</button>
\ No newline at end of file
+<button (click)="changeInputStatus()">{{ isInputEnable ? "Disable" : "Enable" }}</button><input type="text"
+	placeholder="Name" [disabled]="!isInputEnable" (keyup)="handleKeyUp($event)">
+<p *ngIf="isInputEnable">{{ "greeting " + greetingText }}</p>
```

Comment:
```suggestion
<button (click)="changeInputStatus()">
     {{ isInputEnable ? "Disable" : "Enable" }}
</button>
<input type="text" placeholder="Name" [disabled]="!isInputEnable" (keyup)="handleKeyUp($event)">
<p *ngIf="isInputEnable">"greeting, " {{ greetingText }}</p>
```
you can move outside the "greeting" string because it will be whenever